movie/src/components/HumbergerNavbar/index.jsx

```diff
@@ -0,0 +1,64 @@
+import "./HumbergerNavbar.css";
+import NavbarPopup from "../Features/NavbarPopup";
+import mobileList from "../../data/Mobile.json";
+import tvShow from "../../data/TvShow.json";
+import more from "../../data/More.json";
+import SearchIcon from "@mui/icons-material/Search";
+import AddIcon from "@mui/icons-material/Add";
+
+const HumbergerNavbar = ({ onSamsung }) => {
```

Comment:
ughhh... what is `onSamsun`? what happened is the user uses iPhone? 😅

Reply:
I have changed it to onMobile.